### PR TITLE
Minimally reflect the move to the cfgrammar library.

### PR DESCRIPTION
**Pull request by ltratt**

### CODE REVIEW

File: src/lib/mod.rs [hunk 1]
```diff
@@ -32,6 +32,7 @@
 
 #![feature(try_from)]
```
Comment (snim2) [inline src/lib/mod.rs:34]: Extraneous blank line?

**Comment by snim2** [comment]
Looks good, apart from that minor comment.

File: src/lib/mod.rs (hunk 1 above)
Comment (ltratt) [reply src/lib/mod.rs:34]: Ah, no, this is intentional: it's a feature scoped over the entire crate and not connected to one of the `extern crate` lines below.

Comment (snim2) [reply src/lib/mod.rs:34]: OK, in that case I will merge

**Pull request merged**